fastlane/lib/fastlane/actions/git_commit.rb

```diff
@@ -71,7 +67,7 @@ def self.is_supported?(platform)
 
       def self.example_code
         [
-          'git_commit(path: "./version.txt", message: "Version Bump")',
+          # 'git_commit(path: "./version.txt", message: "Version Bump")',
```

Comment:
Force to disable this otherwise `test_sample_code` is failing...
https://github.com/fastlane/fastlane/blob/master/fastlane/actions/test_sample_code.rb

<img width="766" alt="Screenshot 2021-06-13 at 16 52 12" src="https://user-images.githubusercontent.com/5364500/121814393-2770b280-cc71-11eb-9817-4aebc03ee830.png">

Reply:
CI link: https://app.circleci.com/pipelines/github/fastlane/fastlane/3197/workflows/d8482b80-067e-432e-a849-3706df822cb1/jobs/71148

Reply:
@crazymanish What did you run to get this error? 🤔 I checked out this branch and wasn't getting this 😛

Reply:
Step1: Uncomment below line (remove #)
`# 'git_commit(path: "./version.txt", message: "Version Bump")',`

Step2: Run `bundle exec fastlane validate_docs`

Reply:
Oh, duh! You commented them out 😛